README.md

```diff
@@ -7,32 +7,12 @@ In this project, I will set up a "Hello world" repository. Main goal here is to
 
 ## Built With
 
-- Major languages
-- Frameworks
 - Git, Html & Css
 
-## Getting Started
-
-Remember to document your work in a professional way.
-Create a new GitHub repo.
-Create a new branch according to the GitHub flow rules.
-Add a .gitignore file.
-Set up a linter for HTML and CSS and make sure that any files that are not relevant are ignored by git.
-Add descriptive README file to your project - please use this template but remember to customize it to your project.
-Add 3 files (each one in a separate commit):
-A .html file that includes a header with the text "Hello Microverse!". Use h1 tag for that.
-A .css file that includes one class with styles for your header (make it in your favorite color).
-A test.md file with the text "This file should be ignored by git".
-Make sure that the last file is actually ignored by git and is not present in your GitHub repository.
-Open a pull request.
```

Comment:
- [ ] Good job so far 👏 There are some important sections that are missing her in your README file. As it is in the project requirement, we are supposed to use this [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) in order to write README for our project. I appreciate that you have added some content so far but It would be better if all sections in the template are present here in your README file even if you can add brief content in each 😉 so please adjust on that 🏃